code/modules/projectiles/projectile/bullettypes.dm

```diff
@@ -1,197 +1,246 @@
-#define HV_MULT 1.2
-//Low damage pistols and smgs
-#define damage_32 20
-#define damage_9mm 22
-#define damage_10mm 24
-#define damage_45 26
+//Damage multipliers
+#define HIGH_VELOCITY_MULTIPLIER 1.2 //Determines the damage, agony, & armor_penetration values from base damage & armor_penetration.
+#define RUBBER_DAMAGE_MULTIPLIER 0.15 //Determines the damage value from base damage.
+#define RUBBER_AGONY_MULTIPLIER 0.85 //Determines the agony value from base damage.
+#define RUBBER_PENETRATION_MULTIPLIER 0 //Determines the armor_penetration value from base armor_penetration.
+#define PRACTICE_MULTIPLIER 0.1 //Determines damage, armor_penetration, & agony values from base damage.
+//Step delays //Default value is 1. Lower value makes bullet go faster, higher value makes bullet go slower.
+#define HIGH_VELOCITY_STEP_DELAY 0.75
+//Low-caliber pistols and SMGs
+#define ARMOR_PENETRATION_LOW_CALIBER_PISTOL 5
+#define DAMAGE_9MM 22
+#define DAMAGE_10MM 24
+#define DAMAGE_32 20
+#define DAMAGE_45 26
 //Carbines and rifles
-#define damage_556 16
-#define AP_556 25
-#define damage_65 18
-#define AP_65 20
-#define damage_762 20
-#define AP_762 15
+#define DAMAGE_10X24 30
```

Comment:
add in a few linebreaks to seperate the general categories. Just hit enter twice before each comment